src/auth.js

```diff
@@ -4,6 +4,24 @@ const CLIENT_ID =
   '270040816063-djog7f8mpvt4m162ak4n04bjmftg1bhc.apps.googleusercontent.com';
 const client = new OAuth2Client(CLIENT_ID);
 
+const getUser = async (ctx, payload) => {
+  let [user] = await ctx.app.models.User.query()
+    .where('auth_id', payload.sub);
+
+  if (!user) {
+    user = await ctx.app.models.User.query()
+      .insert({
+        auth_id   : payload.sub,
```

Comment:
I'm not sure `.sub` is actually the right thing to be checking against

Reply:
I think it is: https://developers.google.com/identity/sign-in/web/backend-auth